src/tapis-ui/src/components/Systems/index.ts

```diff
@@ -1,3 +1 @@
-import Systems from './Systems';
```

Comment:
Rename the top level component folder lowercase systems. So 

- systems/SystemsList etc

Reply:
This is weird. I wonder if it'll change when. I push up the changes, but this file should be deleted.